Doc/reference/datamodel.rst

```diff
@@ -436,6 +436,11 @@ Mappings
       additional examples of mapping types, as does the :mod:`collections`
       module.
 
+      .. versionchanged:: 3.7
+         Dictionary order is guaranteed to be insertion order. Note that updating a key
```

Comment:
I feel like something is missing here, please ignore if this is the common usage (I'm not a native speaker). 
```suggestion
         The dictionary order is guaranteed to be the same as the insertion order. Note that updating a key
```